js/views/display-payment-address.js

```diff
@@ -225,9 +244,22 @@ app.views.DisplayPaymentAddress = (function() {
 			paymentTransaction.save({confirmed: true});
 		},
 
+		reRenderQrCode: function() {
```

Comment:
Why re render qr code when windows resize? is it expected the window to be resized?

Reply:
If the device is rotated, a resize event is fired and the QR code needs to be re-rendered.